packages/slate/test/models/node/get-closest/matching-current-using-path.js

```diff
@@ -0,0 +1,37 @@
+/** @jsx h */
+
+import h from '../../../helpers/h'
+
+export const input = (
+  <value>
+    <document>
+      <quote key="a" {...{ className: 'slate-node' }}>
+        <paragraph key="b" {...{ className: 'slate-node' }}>
+          <text key="c">one</text>
+        </paragraph>
+      </quote>
+      <paragraph key="d" {...{ className: 'slate-node' }}>
```

Comment:
Also no need to include `key=` on all elements here. We only need it on the specific node we're targeting in the test itself. That way we reduce maintenance burden.

Reply:
Sounds good!